gr-blocks/lib/rotator_cc_impl.cc

```diff
@@ -52,6 +60,19 @@ void rotator_cc_impl::set_phase_inc(double phase_inc)
     d_r.set_phase_incr(exp(gr_complex(0, phase_inc)));
 }
 
+void rotator_cc_impl::handle_phase_inc_msg(pmt::pmt_t msg)
+{
+    if (pmt::is_pair(msg)) {
+        pmt::pmt_t offset    = pmt::car(msg);
+        pmt::pmt_t phase_inc = pmt::cdr(msg);
+        if (pmt::is_uint64(offset) && pmt::is_real(phase_inc)) {
+            d_idx_next_inc_update = pmt::to_uint64(offset);
+            d_next_phase_inc      = pmt::to_double(phase_inc);
+            d_inc_update_pending  = true;
```

Comment:
We'll need a queue of outstanding phase inc changes. Here, sending multiple messages will overwrite the earlier ones.

Reply:
This is still missing. Will complete on my next spare time slot :)

Reply:
I've just had a chance to add this. Used an `std::queue`. Let me know if OK.